hash/rx78.xml

```diff
@@ -2,6 +2,82 @@
 <!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
 <!--
 license:CC0
+
+The Bandai RX-78 Gundam has two slots where you can plug in a ROM cart or an additional device, like a
+printer interface. The system was capable of loading software from these ROM carts, or from a cassette
+tape through audio line-in ports to use with tape recorders; these ports were present in the BS Basic,
+Animation Graphics, Creative Graphics, Japanese Word Processor and Z80 Assembler carts. The system could
+use up to two carts or devices at the same time.
+
+There were a total of 20 games released for the system, plus educational and productivity software, and peripheral
+devices. One additional game and three expansion cards were announced but never released. The official translations
+printed in the game carts or tapes is used below where available. Names in brackets are unofficial translations.
```

Comment:
Grammar - should say “are used”

Reply:
Er, wait, I didn't meant to tag this as resolve conversation. I'll fix the files, I just can't figure out how to, heh.

Reply:
This PR was merged. Fixes can't be applied to it anymore, you'll just need to do a new PR.